src/chunk.js

```diff
@@ -322,11 +324,12 @@ class Chunks {
       }
     });
 
-    // DO NOT SUBMIT: do we still need a viewer at all?
-    /** @protected {?./service/viewer-interface.ViewerInterface} */
-    this.viewer = null;
-    Services.viewerPromiseForDoc(ampDoc).then(viewer => {
-      this.viewer = viewer;
+    /** @protected {boolean} */
+    this.coreReady = false;
```

Comment:
Could this be private to the file? At least use the naming convention for minification

Reply:
Done